patterns/hero-overlapped-book-cover-with-links.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Title: Hero Overlapped Book Cover with Links
+ * Slug: twentytwentyfive/hero-overlapped-book-cover-with-links
+ * Categories: banner, featured
+ * Description: A hero with an overlapped book cover and links.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:cover {"isUserOverlayColor":true,"minHeight":818,"customGradient":"linear-gradient(180deg,rgb(251,250,243) 51%,rgb(255,255,255) 51%)","contentPosition":"center center","align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}},"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80"}}},"textColor":"contrast","layout":{"type":"constrained"}} -->
+<div class="wp-block-cover alignfull has-contrast-color has-text-color has-link-color" style="padding-top:var(--wp--preset--spacing--80);padding-bottom:var(--wp--preset--spacing--80);min-height:818px">
+	<span aria-hidden="true" class="wp-block-cover__background has-background-dim-100 has-background-dim has-background-gradient" style="background:linear-gradient(180deg,rgb(251,250,243) 51%,rgb(255,255,255) 51%)"></span>
+	<div class="wp-block-cover__inner-container">
+		<!-- wp:group {"align":"full","layout":{"type":"constrained","justifyContent":"center","contentSize":"1124px"}} -->
```

Comment:
Not sure if we should set the content size for this one, I'll defer to @beafialho

Reply:
This content size is also comes from Figma design.

Reply:
Apologies for the confusion @Arkenon. The designs in Figma serve as the overall vision for the theme, though as they translate to shipping responsive code, some adjustments organically need to happen. For those, please rely on [my comment below](https://github.com/WordPress/twentytwentyfive/pull/309#issuecomment-2356178148).